pkg/actor/cluster_restart.go

```diff
@@ -239,7 +189,7 @@ func (r *clusterRestart) rollingSts(ctx context.Context, sts *appsv1.StatefulSet
 		}
 
 		// wait for all replicas to be up
-		healthChecker.Probe(ctx, l, "between restarting pods")
+		healthChecker.Probe(ctx, l, "between restarting pods", int(partition))
```

Comment:
Because we wait for the pod to be ready `WaitUntilStatefulSetIsReadyToServe` do we actually need to sleep for a minute? If we do, I'd rather wait here and keep the probe better scoped to what it's trying to accomplish

Reply:
Yes... we need this, because we know that the terminationGracePeriodSeconds is 60 seconds, and has no point to make requests